Sources/BraintreeCore/Analytics/BTAnalyticsService.swift

```diff
@@ -52,18 +50,18 @@ class BTAnalyticsService: Equatable {
     /// - Parameter eventName: String representing the event name
     func sendAnalyticsEvent(
         _ eventName: String,
-        errorDescription: String? = nil,
         correlationID: String? = nil,
+        errorDescription: String? = nil,
         payPalContextID: String? = nil,
         linkType: String? = nil
     ) {
         DispatchQueue.main.async {
-            self.payPalContextID = payPalContextID
             self.enqueueEvent(
                 eventName,
-                errorDescription: errorDescription,
                 correlationID: correlationID,
-                linkType: linkType
+                errorDescription: errorDescription,
+                linkType: linkType,
+                payPalContextID: payPalContextID
```

Comment:
These params are alphabetized in `FPTIBatchData.Event` but not in our method signatures. Updated so that these are in alphabetical order in both places.